packages/react-composites/src/composites/CallWithChatComposite/ChatButtonWithUnreadMessagesBadge.tsx

```diff
@@ -38,6 +39,23 @@ export const ChatButtonWithUnreadMessagesBadge = (props: ChatButtonWithUnreadMes
   const [unreadChatMessagesCount, setUnreadChatMessagesCount] = useState<number>(0);
 
   const baseIcon = props.showLabel ? regularIcon : filledIcon;
+  const callWithChatStrings = useCallWithChatCompositeStrings();
+
+  const numberOfMsgToolTip =
+    props.strings?.tooltipOffContent && unreadChatMessagesCount > 0
+      ? props.strings?.tooltipOffContent +
+        ' (' +
```

Comment:
This shouldn't use string building like this but instead the string in the strings.json should be something like `Show chat ({unreadMessagesCount} unread)` then use _formatString function. See other string building examples in the repo for how this is done such as the readBy tooltip

Reply:
will do!